index.js

```diff
@@ -66,16 +65,16 @@ queue.on('job failed', function(){
 queue.watchStuckJobs();
 
 process.once('uncaughtException', function(err){
-  queue.shutdown(function(err2){
+  queue.shutdown(9500, function(err2){
```

Comment:
having the callback method be the last argument is the newer syntax. https://github.com/Automattic/kue#graceful-shutdown
So the version in master is the old syntax and this should be correct.